examples/ucp_client_server.c

```diff
@@ -887,12 +888,10 @@ memory_allocator_init(ucp_context_h context,
                     sizeof(ucp_worker_memory_allocator_obj_t));
     ucs_mpool_params_t mp_params;
     ucs_status_t status;
-    size_t buffer_size;
+    size_t buffer_size = ALLOCATOR_PAYLOAD_LENGTH;
```

Comment:
1. Unneeded init, in case `sig`. Maybe ternary?
2. Place initialized vars above uninitialized.

Reply:
It's possible, but also it is good to keep optional features like user allocator, signature easy to localize, e.g. all signature related stuff under `if (sig) {}`